pepper-apis/src/main/java/org/broadinstitute/ddp/model/activity/instance/FormResponse.java

```diff
@@ -36,15 +36,28 @@ public FormResponse(@ColumnName("instance_id") long id,
                         @ColumnName("activity_id") long activityId,
                         @ColumnName("activity_code") String activityCode,
                         @ColumnName("activity_version_tag") String activityVersionTag,
+                        @ColumnName("is_hidden") Boolean isHidden,
+                        @ColumnName("section_index") Integer sectionIndex,
                         @Nested ActivityInstanceStatusDto latestStatus) {
         super(ActivityType.FORMS, id, guid, participantId, isReadonly, createdAt, firstCompletedAt,
-                parentInstanceId, parentInstanceGuid, activityId, activityCode, activityVersionTag, latestStatus);
+                parentInstanceId, parentInstanceGuid, activityId, activityCode, activityVersionTag, isHidden, sectionIndex,
+                latestStatus);
     }
 
+    /**
+     * Get list of answers.
+     * @deprecated An answer can be looked up by questionStableId instead of streaming it.
+     *       The choice of using a List of answers was a legacy design decision -
+     *       there should only be one answer per question.
+     */
+    @Deprecated
```

Comment:
I think there's a misunderstanding here. This method should not be deprecated. When I said "legacy design decision", I was not talking about this method -- what I meant was that if you looked at the `Question` class it has a `List<Answer>`. Question should have one answer, so this `List<Answer>` was an old design decision (and no, we do not need to refactor that right now!).

And note that this `getAnswers()` method does NOT return a list of answers for a single question, rather it returns all the answers for all questions in the activity instance. Which is why I suggested not to use this one -- since we want the answer for a single question, we should use `getAnswers(String questionStableId)` instead.

Sorry if I've caused any confusions! Can you please remove the `@Deprecated` annotation?